admin-ui/plugins/admin/redux/actions/types.js

```diff
@@ -60,3 +60,14 @@ export const GET_LICENSE_DETAILS = 'GET_LICENSE_DETAILS'
 export const GET_LICENSE_DETAILS_RESPONSE = 'GET_LICENSE_DETAILS_RESPONSE'
 export const UPDATE_LICENSE_DETAILS = 'UPDATE_LICENSE_DETAILS'
 export const UPDATE_LICENSE_DETAILS_RESPONSE = 'UPDATE_LICENSE_DETAILS_RESPONSE'
+
+// User Management
```

Comment:
User Management types should not be present in `admin` plugin. Plugins should be independent of each other.

Reply:
removed